app/controllers/users_controller.rb

```diff
@@ -4,7 +4,8 @@ def create
     @user.email_confirmation_token = Clearance::Token.new
 
     if @user.save
-      redirect_back_or url_after_create
+      UserMailer.confirmation_email(@user).deliver_now # should be done in a background job when possible
```

Comment:
Please, update the comment to be a proper `TODO: `

Reply:
Done